include/OpenKit/AbstractOpenKitBuilder.h

```diff
@@ -26,6 +26,7 @@
 
 #include <cstdint>
 #include <memory>
+#include <string>
```

Comment:
It is not obvious why the <string> include is required

Reply:
Because there are a lot of fields in the `AbstractOpenKitBuilder` which are of type std::string

Previously <string> was included, where it was not required. I tried to clean up the headers a bit. :)